addons/mail/static/src/discuss/call/common/call_participant_card.js

```diff
@@ -209,31 +209,14 @@ export class CallParticipantCard extends Component {
         this.env.bus.trigger("RTC-SERVICE:PLAY_MEDIA");
     }
 
-    /**
-     * @param {Event} ev
-     */
-    onContextMenu(ev) {
-        markEventHandled(ev, "CallParticipantCard.clickVolumeAnchor");
-        if (this.popover.isOpen) {
-            this.popover.close();
-            return;
-        }
-        if (!this.contextMenuAnchorRef?.el) {
-            return;
-        }
-        this.popover.open(this.contextMenuAnchorRef.el, {
-            rtcSession: this.rtcSession,
-        });
-    }
```

Comment:
Should keep function for stable policy.
The `this.popover` can be kept too, both of which are unused and are flagged `/** @deprecated */`